src/EnergyPlus/DataSizing.cc

```diff
@@ -477,7 +476,7 @@ void GetCoilDesFlowT(EnergyPlusData &state,
         DDAtTotPeak = SysSizPeakDDNum(SysNum).TotCoolPeakDD;
         TimeStepAtTotPeak = SysSizPeakDDNum(SysNum).TimeStepAtTotCoolPk(DDAtTotPeak);
 
-        if (PeakLoadType == TotalCoolingLoad) {
+        if (SysSizInput(SysNum).peakLoad == PeakLoad::TotalCooling) {
```

Comment:
This still has the incorrect `SysNum`, but it should throw a conflict when #9720 is merged.